### DISCUSSION TITLE
Actualisation de scène

### DISCUSSION THREAD

**Discussion by Guillaume31a** [witsa/synapps#676] [Q&A]
Bonjour, je voulais savoir si il était possible d'actualiser une scène lorsqu'on modifie une valeur (par ex sur un graphique je modifie la valeur de X1 et j'aimerais que la courbe bouge automatiquement au lieu de devoir sauvegarder et re cliquer sur la scène).
Faut il ajouter un bout de script en JSON dans l'acteur en question ?
Merci d'avance

**Comment by Guillaume31a** [comment]


**Reply by Raphwit** [reply]
Bonjour, est-ce possible d'avoir un peu plus de précision, quel est ce "X1" mentionné, également si je comprend le comportement souhaité est uniquement pour l'édition ?

**Reply by Guillaume31a** [reply]
Bonjour, j'aimerais vous envoyer le code que j'ai réalisé en message perso pour des soucis de confidentialité

**Reply by Raphwit** [reply]
Oui, vous pouvez l'envoyer à cette adresse si vous souhaitez : raphael.bolier@wit.fr
